### DISCUSSION TITLE
Second G7 sensor = mismatch

### DISCUSSION THREAD

**Discussion by Unrealsweb** [NightscoutFoundation/xDrip#2828] [General]
It is known that sensors can deliver erratic and incorrect values in the first 12-48 hours due to the injection trauma. Here it is advisable to use the 12h transition period so that the tissue can get used to the sensor. The following graphic explains the procedure:
![image](https://github.com/NightscoutFoundation/xDrip/assets/6748067/9c3e15aa-de0e-49d9-aecc-c8b92fbb1ff3)

Sensor A is set
After 10 days the grace period of 12 hours starts and exactly from this time sensor B is set but not activated in the app.
After these 12 hours, sensor A no longer provides any values and sensor B is activated in the app.
This method has several advantages:

Tissue can get used to the sensor and it will then provide more reliable values.
No time without sensor values
Runtime of a sensor is still 10 days

This works with the original and the BYOD app

Unfortunately not with Xdrip, as soon as the second sensor is set (without entering the ID in Xdrip), Xdrip no longer receives anything.

![image](https://github.com/NightscoutFoundation/xDrip/assets/6748067/ef960aae-256a-46a3-932f-9e087747d8d2)
![image](https://github.com/NightscoutFoundation/xDrip/assets/6748067/0b95cb88-ec50-456a-b73b-146d994829b7)

**Comment by Navid200** [comment]
Is 9562 the first transmitter ID or the second?

**Reply by Unrealsweb** [reply]
First

**Comment by jamorham** [comment]
@Unrealsweb I'm surprised you're having this problem as it should lock on to the correct transmitter. Can you enable OB1 debugging https://github.com/NightscoutFoundation/xDrip/wiki/Enabling-Debugging-for-OB1-collector and if you're experiencing the problem now then submit logs once it has time to gather some data with the debugging enabled.

**Reply by Unrealsweb** [reply]
I'll post the logs next time I change the sensor

**Reply by jamorham** [reply]
I think the May 14th Nightly may resolve this issue for you, but it should have automatically recovered before, in that after a period of time the mismatch would go away and things would have continued working.

**Reply by Unrealsweb** [reply]
Problem no longer exists :-)
